content/influxdb/v2.4/reference/glossary.md

```diff
@@ -506,6 +506,14 @@ Related entries: [aggregator plugin](#aggregator-plugin), [collection interval](
 An entity comprising data on a server (or virtual server in cloud computing).
 <!-- An instance in an InfluxDB Enterprise cluster may scale across multiple servers or nodes in a network. -->
 
+{{% oss-only %}}
```

Comment:
@jstirnaman or @sanderson 
Can I use the oss-only shortcode in this manner? 

Doing this causes my page to look like the following:
![image](https://user-images.githubusercontent.com/101659702/188040969-3d96b795-8436-4b32-89b0-b29e243a035e.png)

Reply:
@sunbryely-influxdata Add an empty line after the opening shortcode tag. It should render correctly.